pyrdp/mitm/RDPMITM.py

```diff
@@ -239,11 +239,13 @@ def doClientTls(self):
 
         # Add unknown packet handlers.
         ntlmSSPState = NTLMSSPState()
-        if self.state.ntlmCatch:
-            self.client.segmentation.addObserver(NLAHandler(self.client.tcp, ntlmSSPState, self.getLog("ntlmssp"), True))
-        else:
-            self.client.segmentation.addObserver(NLAHandler(self.server.tcp, ntlmSSPState, self.getLog("ntlmssp")))
-            self.server.segmentation.addObserver(NLAHandler(self.client.tcp, ntlmSSPState, self.getLog("ntlmssp")))
+        self.client.segmentation.addObserver(NLAHandler(self.client.tcp, ntlmSSPState, self.getLog("ntlmssp")))
```

Comment:
Something here puzzles me. We went from `self.server.tcp`, `self.client.tcp` to `self.client.tcp`, `self.client.tcp`. I'm not sure this is accurate.

Reply:
Sorry for that, copy&paste mistake. Fixed on 367ee65.